portfolio/src/main/webapp/style.css

```diff
@@ -16,15 +16,43 @@ a {
 }
 
 a:hover{
+  color: #1c1c1c;
   text-decoration: underline rgb(195, 195, 195);
   text-decoration-thickness: 5px;
-  color: #1c1c1c;
 }
 
-.centered{
+.centered {
   margin: auto;
-  width: 100%;
   text-align: center;
+  width: 100%;
+}
+
+input,
+textarea {
+  background-color: #f7f7f7;
+  border: 0px;
+  color: gray;
+  border-radius: 5px;
+  margin: 5px 0px 5px 0px;
```

Comment:
This would be simpler: `margin: 5px 0px`

Reply:
Fixed!